docs/en/quick-answers/troubleshooting.md

```diff
@@ -10,23 +10,78 @@ Troubleshooting
+- [TankError: Tried to resolve a path from the template](https://community.shotgridsoftware.com/t/unable-to-save-file-in-a-specific-task/9686)
+- [[ERROR] Attempted to communicate without completing encryption handshake.](https://community.shotgridsoftware.com/t/shotgun-desktop-browser-integration/3574)
+- [SSLError: [Errno 8] _ssl.c:504: EOF occurred in violation of protocol](https://community.shotgridsoftware.com/t/sslerror-in-nuke-10-5/9299)
+- [[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate](https://community.shotgridsoftware.com/t/certificate-verify-failed-error-on-windows/8860)
+- [sgtk:descriptor:path?linux_path=None&mac_path=None&windows_path=D\Program%Files\{% include product %}\Configurations does not point to a valid bundle on disk!](https://community.shotgridsoftware.com/t/first-time-setting-up-shotgun-and-i-have-this-error/9384)
+- [TankInitError: You are loading the Toolkit platform from the pipeline configuration located in](https://community.shotgridsoftware.com/t/tankiniterror-loading-toolkit-platform-from-a-different-project/9342)
+- [[ ERROR] [PROXY] Error calling __commands::unreal_engine](https://community.shotgridsoftware.com/t/error-launching-ue4-from-shotgun/8938)
+- [[ERROR publish_creation] <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:726)>](https://community.shotgridsoftware.com/t/ssl-certificate-error-on-sgtk-util-regiter-publish/3291)
+- [Error: {% include product %} tk-maya: An exception was raised from Toolkit](https://community.shotgridsoftware.com/t/custom-app-args/8893)
+- [Tk-desktop console silently ignoring errors](https://community.shotgridsoftware.com/t/tk-desktop-console-silently-ignoring-errors/8570)
+- [TankError: Cannot resolve template data for context](https://community.shotgridsoftware.com/t/tank-folder-creation/8674)
+- [Failed to change work area - Error during execution of MEL script](https://community.shotgridsoftware.com/t/new-file-maya-action-error/8225)
+- [‘ascii’ codec can’t decode byte 0x97 in position 10: ordinal not in range](https://community.shotgridsoftware.com/t/ascii-problem/7688)
+- [TankError: Tried to resolve a path from the template](https://community.shotgridsoftware.com/t/error-when-resolving-path-from-template-at-new-file-open-missing-fields/6468)
+- [ERROR 18:13:28.365:Hiero(34236): Error! Task type](https://community.shotgridsoftware.com/t/cant-get-shotgun-toolkit-to-work-with-nuke-studio-config-default2/4586)
+- [Cannot find procedure "MTsetToggleMenuItem"](https://community.shotgridsoftware.com/t/tk-maya-cannot-find-procedure-mtsettogglemenuitem/4629)
+- [Could not resolve row id for path!](https://community.shotgridsoftware.com/t/how-to-troubleshoot-folder-creation-errors/3578)
+- [Database concurrency problems: The path <PATH> is already associated with {% include product %} entity <ENTITY>](https://community.shotgridsoftware.com/t/how-to-troubleshoot-folder-creation-errors/3578)
```

Comment:
@drewavisautodesk any tips on how to get this to not escape the error message content? <>

Reply:
You could try putting them in backticks: `<ENTITY>`

Or if that doesn't work, use HTML entities: &lt;ENTITY&gt;

Reply:
Of course neither of those actually show the markdown I used... I'll send you them in slack

Reply:
Thanks!

Reply:
I wonder if the {% include product %} in the link text will resolve correctly?  I was having issues with the include working in links/headings